specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/examples/P2SVpnGatewayGetConnectionHealth.json

```diff
@@ -0,0 +1,48 @@
+{
+  "parameters": {
+    "gatewayName": "p2sVpnGateway1",
+    "resourceGroupName": "rg1",
+    "api-version": "2019-04-01",
+    "subscriptionId": "subid"
+  },
+  "responses": {
+    "200": {
```

Comment:
Please also add entry for 202 response code

Reply:
Done.